js/components/Header.js

```diff
@@ -0,0 +1,36 @@
+const headerToken = `<header class="header">
+        <div class="header-cards">
+          <a href=""><div class="card" id="card1">
+            <p>Electronics</p>
+            <i class="fa-solid fa-laptop"></i>
+          </div></a>
+          <a href=""><div class="card" id="card2">
+            <p>Jewelery</p>
+            <i class="fa-solid fa-gem"></i>
+          </div></a>
+          <a href=""><div class="card" id="card3">
+            <p>Men's clothing</p>
+            <i class="fa-solid fa-user-tie"></i>
+          </div></a>
+          <a href=""><div class="card" id="card4">
+            <p>Women's clothing</p>
+            <i class="fa-solid fa-shirt"></i>
+          </div></a>
+        </div>
```

Comment:
use this api to get all categories 
fetch('https://fakestoreapi.com/products/categories')
            .then(res=>res.json())
            .then(json=>console.log(json))
  make things dynamic not static so if we add a new catogry it will be added to the header as well